CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ pkg_check_modules(ZLIB REQUIRED IMPORTED_TARGET zlib)
 pkg_check_modules(EXIF REQUIRED IMPORTED_TARGET libexif)
 pkg_check_modules(AVUTIL REQUIRED IMPORTED_TARGET libavutil)
 pkg_check_modules(SWSCALE IMPORTED_TARGET REQUIRED libswscale)
+pkg_check_modules(OPENSLIDE IMPORTED_TARGET REQUIRED openslide)
```

Comment:
It looks like this might be a dependency not everyone has/needs. Can you make it optional similar to the video modes ?

Reply:
(solved)